python/integration_tests/test_consumer_rebalancing.py

```diff
@@ -79,27 +80,18 @@ def test_tasks_written_once_during_rebalancing() -> None:
     )
     random.seed(42)
 
-    # Ensure topic has correct number of partitions
-    if not check_topic_exists(topic_name):
-        print(
-            f"{topic_name} topic does not exist, creating it with {num_partitions} partitions"
-        )
-        create_topic(topic_name, num_partitions)
-    else:
-        print(
-            f"{topic_name} topic already exists, making sure it has {num_partitions} partitions"
-        )
-        update_topic_partitions(topic_name, num_partitions)
+    # Ensure topic exists and has correct number of partitions
+    create_topic(topic_name, num_partitions)
```

Comment:
What happens if the topic already exists?

Reply:
I think this will print an error message and continue, but usually we run this with `make test_rebalance`, it always clears the kafka container, because reset-kafka is a prerequisite